yassinedoghri/php-iconify
# feat: add global icon(…) function + enforce config path to project root

### Subject:


### Description:

This adds a global `icon(string $iconKey, array $attributes)` function to render icons! It's autoloaded via composer's `autoload.files` property.  
This makes it unnecessary integrate to other frameworks: the library can function by itself, offering a great developer experience.

- add example project to test library
- update README.md
- update dependencies to latest

---

- [x] I confirm that I have read the [Contributing Guide](./../CONTRIBUTING.md) and [Security Guide](./../SECURITY.md)
